.grit/patterns/python/py_imports.grit

```diff
@@ -100,7 +100,7 @@ pattern insert_imports() {
     }
 }
 
-pattern is_imported_from($source) {
+pattern imported_from($source) {
```

Comment:
Was it to differentiate from the JavaScript pattern that we made this `is_imported_from`?